packages/changelog-linker/src/Github/GithubApi.php

```diff
@@ -86,7 +86,7 @@ public function getMergedPullRequestsSinceId(int $id, ?string $baseBranch = null
         $mergedPullRequests = $this->filterMergedPullRequests($pullRequests);
 
         // include all
-        if ($id <= 1) {
+        if ($id < 1) {
```

Comment:
should can be $id === 0 ? 

```suggestion
        if ($id === 0) {
```